silabs-multiprotocol/rootfs/etc/s6-overlay/scripts/universal-silabs-flasher-up

```diff
@@ -23,6 +23,7 @@ fi
 if [ -d /sys/devices/platform/soc/fe201800.serial/tty/ttyAMA1 ] && [ "${device}" == "/dev/ttyAMA1" ]; then
     bashio::log.info "Detected Home Assistant Yellow"
     firmware="NabuCasa_RCP_v4.1.3_rcp-uart-hw-802154_115200.gbl"
+    gpio_reset="--yellow-gpio-reset"
```

Comment:
For clarity, explicitly declare the variable above.

Reply:
Sure. Let me know if <https://github.com/home-assistant/addons/pull/2753/commits/28023d18b2eb2f8a701d70986f46860f4e1b080e> works.